content/docs/faq-versioning.md

```diff
@@ -10,12 +10,14 @@ React follows [semantic versioning (semver)](https://semver.org/) principles.
 
 That means that with a version number **x.y.z**:
 
+* When releasing **critical bug fixes**, we make a **patch release** by changing the **z** number (ex: 15.6.2 to 15.6.3).
+* When releasing **new features** or **non-critical fixes**, we make a **minor release** by changing the **y** number (ex: 15.6.2 to 15.7.0).
```

Comment:
I reversed the order of these items so that "critical bug fixes" comes before "non-critical"